tests/bal_business_logic_suite.py

```diff
@@ -308,6 +309,22 @@ def test_returns_unsupported_policy_for_createRelated_for_all_trait_sets(self):
         self.assertListEqual(actual, expected)
 
 
+class Test_hasCapability(FixtureAugmentedTestCase):
+    """
+    Tests that BAL reports expected capabilities
+    """
+
+    def test_when_hasCapability_called_then_expected_capabilities_reported(self):
+        self.assertFalse(self._manager.hasCapability(Manager.Capability.kStatefulContexts))
+        self.assertFalse(self._manager.hasCapability(Manager.Capability.kCustomTerminology))
+        self.assertFalse(self._manager.hasCapability(Manager.Capability.kDefaultEntityReferences))
+
+        self.assertTrue(self._manager.hasCapability(Manager.Capability.kResolution))
+        self.assertTrue(self._manager.hasCapability(Manager.Capability.kPublishing))
+        self.assertTrue(self._manager.hasCapability(Manager.Capability.kRelationshipQueries))
+        self.assertTrue(self._manager.hasCapability(Manager.Capability.kExistenceQueries))
```

Comment:
[fix] Missing tests for mandatory capabilities

Reply:
Stupidly did this as an amend, sorry, bad habit. This diff shows the changes well enough though : https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/compare/f443089..724cc801ba19b132de701dfc0d525d6fd24d7fe1

Reply:
Actually, `from openassetio_manager_bal import BasicAssetLibraryInterface` works at the top of the `bal_business_logic_suite.py`, thanks to the plugin system.  Still won't help with `pylint`, but is a bit less ugly.

Reply:
It needs to be `from openassetio_manager_bal.BasicAssetLibraryInterface import BasicAssetLibraryInterface`  but otherwise, sure.

https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/pull/79/commits/4f58e94480fa99332c62de91725c34071cc404e3

Reply:
Ah yeah, silly Python. :+1: